artifactory_test.go

```diff
@@ -326,6 +327,60 @@ func TestArtifactoryDirectoryCopyUsingWildcard(t *testing.T) {
 	cleanArtifactoryTest()
 }
 
+func TestArtifactoryCopyFilesNameWithBracket(t *testing.T) {
```

Comment:
Shouldn't the function name be **TestArtifactoryCopyFilesNameWithParenthesis**?
Same goea for the *TestArtifactoryUploadFilesNameWithBracket* and *TestArtifactoryDownloadFilesNameWithBracket* functions.

Reply:
Done.